packages/contracts-bedrock/README.md

```diff
@@ -267,13 +268,36 @@ Maintaining a consistent code style makes code easier to review and maintain, ul
 
 ## Deployment
 
-The smart contracts are deployed using `foundry` with a `hardhat-deploy` compatibility layer. When the contracts are deployed,
-they will write a temp file to disk that can then be formatted into a `hardhat-deploy` style artifact by calling another script.
+The smart contracts are deployed using `foundry`. The `DEPLOYMENT_OUTFILE` env var will determine the filepath that the
+deployment artifact is written to on disk after the deployment. It comes in the form of a JSON file where keys are
+the names of the contracts and the values are the addresses the contract was deployed to.
 
-The addresses in the `deployments` directory will be read into the script based on the backend's chain id.
-To manually define the set of addresses used in the script, set the `CONTRACT_ADDRESSES_PATH` env var to a path on the local
-filesystem that points to a JSON file full of key value pairs where the keys are names of contracts and the
-values are addresses. This works well with the JSON files in `superchain-ops`.
+The `DEPLOY_CONFIG_PATH` is a filepath to a deploy config file, see the `deploy-config` directory for examples and the
+[DeployConfig](https://github.com/ethereum-optimism/optimism/blob/develop/op-chain-ops/genesis/config.go) definition for
+descriptions of the values.
+
+```bash
+DEPLOYMENT_OUTFILE=deployments/artifact.json \
+DEPLOY_CONFIG_PATH=<PATH_TO_MY_DEPLOY_CONFIG> \
```

Comment:
Can we add some more details on what exactly this will deploy? e.g. does it deploy a new SuperchainConfig or point to an existing one? what about pointing to existing MCP implementation contracts?

Reply:
It deploys a new `SuperchainConfig`, this is the exact deploy script that people use today. We have no logic for pointing to existing MCP implementations in the deploy script so I would like to consider that out of scope for this. This is a straight improvement over our docs site (which doesn't work with `develop`). The work to make it do the MCP setup is minimal but I don't have time to do that now, I think it could be done in a single day

Reply:
Ah, yea sorry I'm not saying we should make it MCP compatible in this PR, just suggesting we document that this command does deploy new superchain configs and also deploys new implementations (i.e. not MCP compatible )

Reply:
Ah ok sorry, I understand what you mean, I can add some details about what it does